documentation/hang/step1/using_trace_sigint.md

```diff
@@ -1 +1,61 @@
-// TODO
+# Using SigInt Traces for Call Stacks
+
+`--trace-sigint` CLI option will print a message and stack traces on SIGINT.
+It can be convenient to find out what's the process doing when using shells
+launched the Node.js processes.
+
+> Caveat:
+> `--trace-sigint` CLI option is available since Node.js v13.9.0.
```

Comment:
should we add another caveat that says the process control behavior of `SIGINT` is un-altered otherwise? i.e, by default, the process also get terminated after the print?

Reply:
I added another paragraph in the first section mentioning the unchanged behavior. PTAL :)